napari/_qt/qt_about_keybindings.py

```diff
@@ -74,16 +74,8 @@ def change_layer_type(self, text):
 
     def update_active_layer(self, event):
         col = self.viewer.palette['secondary']
-        text = ''
         # Add class and instance viewer keybindings
-        text += get_keybindings_summary(self.viewer.class_keymap, col=col)
-        text += get_keybindings_summary(self.viewer.keymap, col=col)
-
-        layer = self.viewer.active_layer
-        if layer is not None:
-            # Add class and instance layer keybindings for the active layer
-            text += get_keybindings_summary(layer.class_keymap, col=col)
-            text += get_keybindings_summary(layer.keymap, col=col)
+        text = get_keybindings_summary(self.viewer.active_keymap, col=col)
```

Comment:
side note - mainly for me to keep track of - I should have used `color` instead of `col` for the keyword argument here, as `col` is definitely confusing. No need to fix in this PR though